### PR TITLE
Adding tests for checking if the current versions are the latest

### PR DESCRIPTION
**Pull request by tmatsuo**
Now it only checks micro versoins of php56, php70, php71, and nginx.
My plan is to run this on Travis with its Cron feature.

### CODE REVIEW

**Comment by tmatsuo** [comment]
I haven't run it on travis, but it successfully failed (detected our nginx is old) locally.

File: .travis.yml [hunk 1]
```diff
@@ -38,4 +45,4 @@ install:
 - scripts/install_test_dependencies.sh
 
 script:
-- scripts/run_test_suite.sh
+- scripts/check_versions.sh
```
Comment (chingor13) [inline .travis.yml]: So travis will no longer run our image tests?

Comment (tmatsuo) [inline .travis.yml]: We used to use Travis, but migrated to CircleCI. We haven't been using Travis for a long time.

**Comment by tmatsuo** [comment]
@chingor13 Thanks,I removed the `cloudbuild.yaml`, PTAL

File: .travis.yml (hunk 1 above)
Comment (chingor13) [inline .travis.yml]: Ok, so we'll start using it again just for the version checks? Looks like it's a new beta feature of Travis that Circle doesn't have.

Comment (tmatsuo) [inline .travis.yml]: Ya

**Pull request merged**